### PR TITLE
cleanup more usages of EnvoyFilter

### PR DESCRIPTION
**Pull request by kyessenov**
- Update to use Telemetry API
- Replace proxy-proto EnvoyFilter
- Note the change in 1.18.

### CODE REVIEW

File: content/en/docs/tasks/security/authorization/authz-ingress/index.md [hunk 1]
```diff
@@ -217,13 +217,13 @@ metadata:
...
+  - applyTo: LISTENER_FILTER
     patch:
-      operation: MERGE
+      operation: INSERT_FIRST
       value:
-        listener_filters:
-        - name: envoy.listener.proxy_protocol
-        - name: envoy.listener.tls_inspector
+        name: proxy_protocol
+        typed_config:
```
Comment (howardjohn) [inline content/en/docs/tasks/security/authorization/authz-ingress/index.md:225]: should we update tests/integration/pilot/gw_topology_test.go to do this method as well

**Review by howardjohn — APPROVED** [review]

File: content/en/docs/tasks/security/authorization/authz-ingress/index.md (hunk 1 above)
Comment (kyessenov) [reply content/en/docs/tasks/security/authorization/authz-ingress/index.md:225]: yeah, it's done in https://github.com/istio/istio/pull/45223

**Comment by ericvn** [comment]
`make snips` needed. Not sure if it will affect tests or not with snippet renumbering.

**Comment by kyessenov** [comment]
@ericvn Updated snip tests. Please review.

File: content/en/docs/tasks/observability/metrics/customize-metrics/index.md [hunk 2]
```diff
@@ -17,24 +17,8 @@ your mesh. For example, dashboards that support Istio include:
...
-supports creating new metrics by name, the expected value expression, and the
-metric type (`counter`, `gauge`, and `histogram`). The `metrics` section
-provides values for the metric dimensions as expressions, and allows you to
-remove or override the existing metric dimensions. You can modify the standard
-metric definitions using `tags_to_remove` or by re-defining a dimension. These
-configuration settings are also exposed as istioctl installation options, which
-allow you to customize different metrics for gateways and sidecars as well as
-for the inbound or outbound direction.
+`requests_total`), but you can also customize them and create new metrics
+using [Telemetry API](/docs/tasks/observability/telemetry/).
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/customize-metrics/index.md:21]:
```suggestion
using the [Telemetry API](/docs/tasks/observability/telemetry/).
```

File: content/en/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 3]
```diff
@@ -18,7 +18,10 @@ This task shows you how to customize the metrics that Istio generates with Telem
 
 Telemetry API can not work together with `EnvoyFilter`. For more details please checkout this [issue](https://github.com/istio/istio/issues/39772).
 
-* Starting with Istio version `1.18`, the stats `EnvoyFilter` will not be installed by default.
+* Starting with Istio version `1.18`, Prometheus `EnvoyFilter` will not be
```
Comment (ericvn) [inline content/en/docs/tasks/observability/metrics/telemetry-api/index.md:21]:
```suggestion
* Starting with Istio version `1.18`, the Prometheus `EnvoyFilter` will not be
```

**Review by ericvn — COMMENTED** [review]
A few minor comments.

**Review by ericvn — APPROVED** [review]

**Pull request merged**